Formula/elixir-ls.rb

```diff
@@ -0,0 +1,34 @@
+class ElixirLs < Formula
+  desc "Language Server and Debugger for Elixir"
+  homepage "https://elixir-lsp.github.io/elixir-ls"
+  url "https://github.com/elixir-lsp/elixir-ls/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "eaa2f5fd4ab9e497889e25419e4409cfaf808daca06259718668a71de8616e3f"
+  license "Apache-2.0"
+
+  depends_on "elixir"
+
+  ENV["MIX_ENV"] = "prod"
+
+  def install
+    system "mix", "local.hex", "--force"
+    system "mix", "local.rebar", "--force"
+    system "mix", "deps.get"
+    system "mix", "compile"
+    system "mix", "elixir_ls.release", "-o", prefix.to_s
+  end
+
+  def caveats
+    <<~EOS
+      By default, elixir-ls will be placed into:
+      /usr/local/opt/elixir-ls
+
+      You may want to add this to your PATH.
+
+      Make sure your editor also loads this PATH.
+    EOS
+  end
+
+  test do
+    assert_predicate prefix/"launch.sh", :exist?
```

Comment:
can we improve the test a bit?

Reply:
I think this might not be possible, unless @axelson corrects me. See https://github.com/asdf-vm/asdf/issues/785